x-pack/plugins/osquery/public/agents/use_osquery_policies.ts

```diff
@@ -13,16 +14,19 @@ import { OSQUERY_INTEGRATION_NAME } from '../../common';
 export const useOsqueryPolicies = () => {
   const { http } = useKibana().services;
 
-  const { isLoading: osqueryPoliciesLoading, data: osqueryPolicies } = useQuery(
+  const { isLoading: osqueryPoliciesLoading, data } = useQuery(
     ['osqueryPolicies'],
     () =>
       http.get(packagePolicyRouteService.getListPath(), {
         query: {
           kuery: `${PACKAGE_POLICY_SAVED_OBJECT_TYPE}.package.name:${OSQUERY_INTEGRATION_NAME}`,
         },
       }),
-    { select: (data) => data.items.map((p: { policy_id: string }) => p.policy_id) }
+    {
+      select: (response) =>
+        uniq<string>(response.items.map((p: { policy_id: string }) => p.policy_id)),
+    }
   );
-
+  const osqueryPolicies = data ?? [];
   return { osqueryPoliciesLoading, osqueryPolicies };
```

Comment:
can we directly return `useQuery`? 
or memoize the return object? otherwise, I think it will rerender every time because do create a new object everytime the hook is executed